src/app/components/mnoe-invoices-list/mnoe-invoices-list.coffee

```diff
@@ -20,31 +20,25 @@
         ctrl.invoices.offset = (page  - 1) * nbItems
         fetchInvoices(nbItems, ctrl.invoices.offset)
 
-    # Server call
-    ctrl.callServer = (tableState) ->
-      search = updateSearch (tableState.search)
-      fetchInvoices(ctrl.invoices.nbItems, ctrl.invoices.offset)
+    ctrl.searchChange = ->
+      if ctrl.invoices.search && ctrl.invoices.search.length > 2
+        fetchInvoices(ctrl.invoices.nbItems, ctrl.invoices.offset)
 
     # Fetch invoices
-    fetchInvoices = (limit, offset, search = ctrl.invoices.search) ->
+    fetchInvoices = (limit, offset, query_param = ctrl.invoices.search) ->
       ctrl.invoices.loading = true
+      search = {}
+      search[ 'where[slug.like]' ] = "#{query_param}%" if query_param
```

Comment:
Previously did the search only apply to the `slug` attr?

Reply:
Yes it is only applied to slug. [Reference](https://github.com/maestrano/mnoe-admin-panel/pull/320/files#diff-36db33466520d50b271273726fd9966fL45)